rgw/v2/lib/s3/write_io_info.py

```diff
@@ -242,6 +282,71 @@ def add_keys_info(self, access_key, bucket_name, key_info):
         )
         self.file_op.add_data(yaml_data)
 
+    def set_key_deleted(self, bucket_name, key_name):
+        """
+        This function to add properties to the yaml
+
+        Parameters:
+            bucket_name: name of the bucket
+            key_name: name of the key
+        """
+        log.info(f"marking key '{key_name}' in bucket '{bucket_name}' as deleted")
+        yaml_data = self.file_op.get_data()
+        access_key_indx = None
+        bucket_indx = None
+        key_indx = None
+        for i, _ in enumerate(yaml_data["users"]):
+            for j, k in enumerate(yaml_data["users"][i]["bucket"]):
+                if k["name"] == bucket_name:
+                    bucket_indx = j
+                    access_key_indx = i
+                    break
+        for i, k in enumerate(
+            yaml_data["users"][access_key_indx]["bucket"][bucket_indx]["keys"]
+        ):
+            if k["name"] == key_name:
+                key_indx = i
+                break
+        yaml_data["users"][access_key_indx]["bucket"][bucket_indx]["keys"][key_indx][
+            "deleted"
+        ] = True
+        self.file_op.add_data(yaml_data)
+
+    def set_keys_deleted_in_bucket_with_prefix(
```

Comment:
Can we have one run with LC script and one with delete object script

Reply:
uday, I have raised seperate PR for lc verify io. This PR is only for delete object, attached the logs of it.